src/commands/stop.ts

```diff
@@ -31,6 +31,25 @@ export default class Stop extends Command {
     const { args } = this.parse(Stop);
     let relistProcesses = false;
 
+    if(args.id === undefined) {
+      // Prompt for process
+      const processes: ProcessDescription[] = await ProcessManager.list();
+
+      if(processes.length === 0) {
+        this.log(Messages.CLI.NO_RUNNING_PROCESS);
+
+        return;
```

Comment:
You must disconnect PM2 otherwise the CLI will hang.

```suggestion
        ProcessManager.disconnect();
        
        return;
```